lambdas/src/apis/collections/controllers/collections.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from 'express'
+import { SmartContentServerFetcher } from '../../../SmartContentServerFetcher'
+
+export async function getStandardErc721(fetcher: SmartContentServerFetcher, req: Request, res: Response) {
+    // Method: GET
+    // Path: /standard/erc721/:contract/:option/:emission
+    const { contract, option, emission } = req.params;
+
+    try {
+        const entities:WearableEntity[] = await fetcher.fetchJsonFromContentServer(`/entities/wearable?pointer=${contract}-${option}`)
```

Comment:
Can we start using the `/deployments` endpoint?
It would be something like `/deployments?entityType=wearable&pointer=${contract}-${option}&onlyCurrentlyPointed=true?fields=metadata`

In a next iteration, it would also be nice to start using the [client](https://github.com/decentraland/catalyst-client/)

Reply:
Same here.